db/cofed.sql

```diff
@@ -51,12 +58,72 @@ GRANT ALL PRIVILEGES ON ALL SEQUENCES IN SCHEMA public TO root;
 
 GRANT ALL PRIVILEGES ON DATABASE cofed TO root;
 
+INSERT INTO coops
+    (email, hashed_pass, coop_name, phone_number, addr, latitude, longitude, website, mission_statement, description_text, profile_pic, insta_link, fb_link)
+VALUES
+    ('test@gmail.com', 'password', 'Test', '123456789',
+        'Berkeley, CA', 37.8715, -123.2730, 'test.com', 'test', 'test', 'test', 'test',
+        'test'),
+    ('test2@gmail.com', 'password2', 'Test2', '123456789',
+        'Berkeley, CA2', 37.8715, -123.2730, 'test.com', 'test2', 'test2', 'test2', 'test2',
+        'test2'),
+    ('test3@gmail.com', 'password3', 'Test3', '123456789',
+        'Berkeley, CA3', 37.8715, -123.2730, 'test.com', 'test3', 'test3', 'test3', 'test3',
+        'test3'),
+    ('test4@gmail.com', 'password4', 'Test4', '123456789',
+        'Berkeley, CA2', 37.8715, -123.2730, 'test.com', 'test2', 'test2', 'test2', 'test2',
+        'test2'),
+    ('test5@gmail.com', 'password5', 'Test5', '123456789',
+        'Berkeley, CA3', 37.8715, -123.2730, 'test.com', 'test3', 'test3', 'test3', 'test3',
+        'test3');
+
+INSERT INTO categories
+    (category_name)
+VALUES
+    ('Organization Structure'),
+    ('Products'),
+    ('Coop Size');
+
 INSERT INTO tags
-    (tag_name)
+    (tag_name, category_id)
+VALUES
+    ('Cooperative' , 1),
+    ('Distributor', 1),
+    ('Producer' , 1),
+    ('Community', 1),
+    ('Delivery', 1),
+    ('Medicinal Fungi' , 2),
+    ('Animal Products', 2),
+    ('Herbs' , 2),
+    ('Spices' , 2),
+    ('1-10', 3),
+    ('10-50', 3),
+    ('50-100', 3);
```

Comment:
great!! let's also add a category `'Services'` with
- advocacy
- education
- food services
- purchasing and acquisition

and also maybe add a 100+ tag for the Coop Size category!

@leebianca lmk if there's any more missing